internal/display/custom_domain.go

```diff
@@ -0,0 +1,97 @@
+package display
+
+import (
+	"strconv"
+
+	"github.com/auth0/auth0-cli/internal/ansi"
+	"gopkg.in/auth0.v5/management"
+)
+
+type customDomainView struct {
```

Comment:
what do you think about and  approach like this to simplify here and there?

```go
type customDomainView struct {
   raw *management.CustomDomain
}

func (v *customDomainView) AsTableRow() []string {
	return []string{
		ansi.Faint(v.raw.ID),
		v.raw.Domain,
		v.raw.Status,
	}
}
```

Reply:
I think keeping a separate view model that does not know about the underlying DTO is something that will make it easier and simpler to decouple the SDK in the future (if the project gets to that stage). Otherwise the data layer gets a (bigger) hard coupling with the view layer.

Reply:
I mean, both layers are already coupled, but doing that will make it worse.